infra/src/main/kotlin/fi/fta/geoviite/infra/split/SplitDao.kt

```diff
@@ -11,6 +13,17 @@ import fi.fta.geoviite.infra.util.*
 import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
 import org.springframework.stereotype.Component
 import org.springframework.transaction.annotation.Transactional
+import java.sql.ResultSet
+
+private fun ResultSet.toSplit(targetLocationTracks: List<SplitTarget>) = Split(
```

Comment:
Olen itse aina karsastanut external funkkareita silloin jos saman voi hoitaa tavallisellakin. Eli mielummin vain 
`private fun toSplit(rs: ResultSet, targetLocationTracks: ...)`. Tuo on siis kyllä ihan mutu-juttu, paljonhan näitä kotlinissa tykätään kylvää mutta itse olen käyttänyt vain silloin kun on oikeasti ollut tarve laajentaa jotain olemassaolevaa api:a (esim tuota jdbc:n).

Myös yleisesti nämä resultset mappaukset on mielipiteitä jakava. Aikanaan todettiin että kirjoitetaan mielummin ne käyttöpaikkaansa jotta nuo columnit on sen sql:n lähellä, sillä niiden täytyy kuitenkin olla 1=1. Toisaalta näitä mappereita on muuallakin pari kyllä irrotettu silloin kun ne on selvästi duplikaatteja. Olisikohan tuohon joku hyvä patterni...